tests/test_today_i_did_bot.py

```diff
@@ -0,0 +1,28 @@
+import pytest
+import os
+from slack_today_i_did.bot_file import TodayIDidBot
+
+MOCK_PERSON = 'dave'
+MOCK_CHANNEL = '#durp'
+MOCK_START_SESSION_TEXT = 'start-session'
+MOCK_TEST_FILE = '.testdata_sessions'
+
+
+def test_session(mocker):
```

Comment:
:art: is this test about parsing a `start-session` message correctly - `test_start_session`? or `test_start_session_command`?